src/commands/devchain.js

```diff
@@ -121,12 +149,31 @@ exports.printResetNotice = (reporter, reset) => {
   }
 }
 
-exports.handler = async ({ reporter, port, reset, verbose, accounts }) => {
-  const task = await exports.task({ port, reset, verbose, reporter, showAccounts: accounts })
+exports.handler = async ({
+  reporter,
+  port,
+  reset,
+  verbose,
+  accounts,
+  apm: apmOptions,
+  silent,
+  debug,
+}) => {
+  const task = await exports.task({
+    port,
+    reset,
+    verbose,
+    reporter,
+    showAccounts: accounts,
+    silent,
+    debug,
+  })
   const { privateKeys, mnemonic } = await task.run()
   exports.printAccounts(reporter, privateKeys)
   exports.printMnemonic(reporter, mnemonic)
   exports.printResetNotice(reporter, reset)
 
+  reporter.info(`ENS instance deployed at ${apmOptions['ens-registry']}\n`)
```

Comment:
This address should always be [`0x5f6f7e8cc7346a11ca2def8f827b7a0b612c56a1`](https://github.com/aragon/aragen/blob/b7452488b0/readme.md#using-snapshots) due to how we [deterministically deploy ENS](https://github.com/aragon/aragen/blob/b7452488b0/scripts/deploy-base#L9) using the same account and nonce each time.